.travis.yml

```diff
@@ -20,10 +20,10 @@ env:
   - TEST_PHP_SQL_PWD=Password123
 
 before_install:
-  - docker pull mcr.microsoft.com/mssql/server:2019-GA-ubuntu-16.04
+  - docker pull mcr.microsoft.com/mssql/server:2019-latest

```

Comment:
This installs the ubuntu version of SQL Server?

Reply:
Yes, please scroll down this [page for your reference ](https://hub.docker.com/_/microsoft-mssql-server)